parser-typechecker/unison/Main.hs

```diff
@@ -1,5 +1,6 @@
 {-# Language OverloadedStrings #-}
 {-# Language PartialTypeSignatures #-}
+{-# Language ScopedTypeVariables #-}
```

Comment:
Comment on this file - I don't think anything interesting here. We create the cache and plumb it everywhere it's needed.